spec/controllers/venues_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe Api::VenuesController, type: :controller do
+  describe 'GET #show' do
+    let(:venue_attributes) { %w[id address description people_capacity] }
+
+    context 'when valid' do
+      let(:venue) { create(:venue) }
+      it 'returns json response with venue' do
+        get :show, params: { id: venue.id }
+        json_response = JSON.parse(response.body)
```

Comment:
You will use this `JSON.parse(response.body)` billion times in near future. next time, or right now create a helper for this operation

Reply:
resolved